doc/nrf/app_dev/device_guides/working_with_nrf/nrf54h/ug_nrf54h20_suit_nordic_bundles.rst

```diff
@@ -0,0 +1,71 @@
+.. _ug_nrf54h20_suit_soc_binaries:
```

Comment:
Please edit also the filename to match the shortcut

Reply:
Done